src/config/main.js

```diff
@@ -2,62 +2,88 @@
 // These settins can then be imported anywhere in the app as they are exported at the botom of the file.
 
-import { jsPsych } from 'jspsych-react'
-import _ from 'lodash'
-import { eventCodes } from './trigger'
-import {init} from '@brown-ccv/behavioral-task-trials'
+import { jsPsych } from "jspsych-react";
+import _ from "lodash";
+import { eventCodes } from "./trigger";
+import { init } from "@brown-ccv/behavioral-task-trials";
+import { getProlificId } from "../lib/utils";
 
 // mapping of letters to key codes
 const keys = {
-	"A": 65,
-	"B": 66,
-	"C": 67,
-	"F": 70,
-	"J": 74,
-	"space": 32
-}
+  A: 65,
+  B: 66,
+  C: 67,
+  F: 70,
+  J: 74,
+  space: 32,
+};
 
 // audio codes
 const audioCodes = {
-	frequency: 100*(eventCodes.open_task - 9),
-	type: 'sine'
-}
+  frequency: 100 * (eventCodes.open_task - 9),
+  type: "sine",
+};
+
+const taskName = "honeycomb template";
 
 // is this mechanical turk?
-const MTURK = (!jsPsych.turk.turkInfo().outsideTurk)
-const AT_HOME = (process.env.REACT_APP_AT_HOME === 'true')
-let IS_ELECTRON = true
+const MTURK = !jsPsych.turk.turkInfo().outsideTurk;
+let PROLIFIC = getProlificId() && !MTURK;
+let IS_ELECTRON = true;
+let FIREBASE = process.env.REACT_APP_FIREBASE === "true";
 
 try {
-	window.require('electron')
+  window.require("electron");
 } catch {
-	IS_ELECTRON = false
+  IS_ELECTRON = false;
 }
 
+// these variables depend on IS_ELECTRON
+// whether or not to ask the participant to adjust the volume
+const VOLUME = process.env.REACT_APP_VOLUME === "true";
+// whether or not to enable video
+const VIDEO = process.env.REACT_APP_VIDEO === "true" && IS_ELECTRON;
+// whether or not the EEG/event marker is available
+const USE_EVENT_MARKER =
+  process.env.REACT_APP_USE_EVENT_MARKER === "true" && IS_ELECTRON;
+// whether or not the photodiode is in use
+const USE_PHOTODIODE =
+  process.env.REACT_APP_USE_PHOTODIODE === "true" && IS_ELECTRON;
+
 // get language file
-const lang = require('../language/en_us.json')
-if (MTURK) { // if this is mturk, merge in the mturk specific language
-  const mlang = require('../language/en_us.mturk.json')
-	_.merge(lang, mlang)
+const lang = require("../language/en_us.json");
+if (!IS_ELECTRON) {
+  // if this is mturk, merge in the mturk specific language
+  const mlang = require("../language/en_us.mturk.json");
+  _.merge(lang, mlang);
 }
 
 const defaultBlockSettings = {
-	conditions: ["a", "b", "c"],
-	repeats_per_condition: 1, // number of times every condition is repeated
-	is_practice: false,
-	is_tutorial: false,
-	photodiode_active: false
-}
+  conditions: ["a", "b", "c"],
+  repeats_per_condition: 1, // number of times every condition is repeated
+  is_practice: false,
+  is_tutorial: false,
+  photodiode_active: false,
+};
 
 // setting config for trials
-const config = init({USE_PHOTODIODE: !AT_HOME && IS_ELECTRON,  USE_EEG: false, USE_ELECTRON: IS_ELECTRON, USE_MTURK: MTURK})
+const config = init({
+  USE_PHOTODIODE: USE_PHOTODIODE,
```

Comment:
the variables that are the same name can use the `{ USE_PHOTODIODE, ... }` syntax

Reply:
which might just be `USE_PHOTODIODE` at this point... should the original declaration of these vars match the config names so that this mapping is cleaner?